src/annotator/guest.coffee

```diff
@@ -165,7 +165,14 @@ module.exports = class Guest extends Delegator
     crossframe.on 'scrollToAnnotation', (tag) =>
       for anchor in @anchors when anchor.highlights?
         if anchor.annotation.$tag is tag
-          scrollIntoView(anchor.highlights[0])
+          event = new CustomEvent('scrollToRange', {
```

Comment:
The [CustomEvent constructor](https://developer.mozilla.org/en/docs/Web/API/CustomEvent#Browser_compatibility) is not available in IE (we support 10 & 11 only), although the event type itself is. I think [this polyfill](https://www.npmjs.com/package/custom-event) might suffice.